### ISSUE TITLE
Content and scrollbar are not displayed properly

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#12]
The content and scrollbar are not displayed properly when the height of the window is greater than the screen resolution on MacOS X 10.6.8 (custom built Shoes).

![mac os x server 10 6-2014-12-28-19-32-33](https://cloud.githubusercontent.com/assets/8844578/5565414/51edf0f6-8ec8-11e4-9f74-c5c854a580d8.png)

```ruby
Shoes.app(width: 800, height: 1400) do
   (1..100).each do |n|
      para "Filling up space #{n}\n"
   end
end
```

**Comment by BackOrder** [comment]
Code sample causing the issue is committed to **bugs** directory in Shoes3 repository, commit 70b694b9535269f618b9a3a463e8bb033643e3d2.

**Comment by ccoupe** [comment]
Gtk appears not to allow a window larger that the screen minus the menu bar, docks etc. Shoes OSX should do the same. It's easy to get the screen size and constrain app->height  to that and the example does display the sample starting at line 5 - depends on the size of the menu bar and dock, the later being user changeable.  There are full screen confusions nearby...

**Comment by ccoupe** [comment]
This was fixed in 3.2.24.
